src/sentry/plugins/sentry_mail/activity/base.py

```diff
@@ -138,7 +138,6 @@ def get_headers(self):
         group = self.group
 
         headers = {
-            'X-Sentry-Team': project.team.slug,
```

Comment:
Talked to Ted about these. I tried to figure out how we'd include multiple values for the same header key without much luck. Since Ted discovered that gmail doesn't even allow filtering on custom headers, we figured it was probably not worth the hassle of customizing this for each recipient and decided to just remove these.